src/mlpack/core/tree/greedy_single_tree_traverser.hpp

```diff
@@ -40,12 +40,19 @@ class GreedySingleTreeTraverser
   //! Get the number of prunes.
   size_t NumPrunes() const { return numPrunes; }
 
+  //! Set value of minBaseCases.
+  void MinBaseCases(size_t baseCases) { minBaseCases = baseCases; }
```

Comment:
I'm really sorry to be picky on this one, especially because I know there is ambiguity in the mlpack code.  I think it is better (or matches more of the mlpack codebase) to allow direct access:

```
//! Set value of minBaseCases.
size_t& MinBaseCases() { return minBaseCases; }
//! Get value of minBaseCases.
size_t MinBaseCases() const { return minBaseCases; }
```

Reply:
No problem, thanks for pointing it out. Will update it soon.